internal/handlers/common/update.go

```diff
@@ -110,6 +120,55 @@ func UpdateDocument(doc, update *types.Document) (bool, error) {
 	return changed, nil
 }
 
+// processCurrentdateFieldExpression changes document according to $currentDate operator.
+// If the document was changed it returns true.
+func processCurrentdateFieldExpression(doc *types.Document, currentDateVal any) (bool, error) {
+	var changed bool
+	var err error
+	currentDateExpression := currentDateVal.(*types.Document)
+
+	now := time.Now()
```

Comment:
I wonder if we should take care of timezones and other things somewhere... I guess, this is enough, but I just want us to be mindful about it :)

Reply:
> I wonder if we should take care of timezones and other things somewhere... I guess, this is enough, but I just want us to be mindful about it :)

Here we have to use UTC. Changed to UTC and tests didn't fail. Interesting.